man/DataImport.Rd

```diff
@@ -0,0 +1,58 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/dettl.R
+\name{DataImport}
+\alias{DataImport}
+\title{Data Import}
+\description{
+Manage data import.
+}
+\section{Methods}{
+
+
+\describe{
+\item{\code{extract}}{
+Run the extract stage of the data import
+\cr\emph{Usage:}\preformatted{extract()}
+}
+\item{\code{transform}}{
+Run the transform stage of the data import
+\cr\emph{Usage:}\preformatted{transform()}
+}
+\item{\code{load}}{
+Run the load stage of the data import
+\cr\emph{Usage:}\preformatted{load(comment = NULL, dry_run = FALSE, force = FALSE)}
+
+\emph{Arguments:}
+\itemize{
+\item{\code{comment}:   Optional comment which will be persisted in the log of the import run in the database.
+}\preformatted{\item{\code{dry_run}:   Whether to run in dry run mode. If TRUE then any database changes will be rolled back. Defaults to FALSE.
```

Comment:
This is one thing I wanted to ask about.

Where is this `\preformatted` tag coming from? I cannot work this out at all. The `man-roxygen/DataImport.R` file generated by `autodoc.R` script looks equivalent to the examples in `vaultr` yet when I generate the documentation here using `devtools::document(roclets=c('rd', 'collate', 'namespace'))` it inserts a `preformatted` block after the first arg. Do you have any ideas why this is? I wonder if I am just missing something.

Reply:
You have `Roxygen: list(markdown = TRUE)` in your DESCRIPTION

Reply:
Nice, I've fixed that and used `\code` where I was using back ticks.